portfolio/src/main/webapp/index.html

```diff
@@ -41,6 +40,18 @@ <h2 class="greeting">Hi, I'm Karol Gutiérrez</h2>
       <button class="center" onclick="changeCat()">Click here to get a random image of some of my cats:</button>
       <br/>
       <img id="cat-image" class="center" src="images/cats/cat_1.jpeg" />
+
+        <!--COMMENTS-->
+        <h2>Comments</h2>
+        <div id="comments-container"></div>
+
+        <!--SUBMIT COMMENT-->
+        <form action="/data" method="POST">
+            <p>Comment here:</p>
+            <textarea name="text-input" placeholder="Enter yout comment here"></textarea>
```

Comment:
Is there a more descriptive name we should use here instead of text-input?  Maybe something like text-comment?